docs/3.0. Authorization Server Setup.md

```diff
@@ -36,6 +36,11 @@ You can get a latest version from the [Keycloak website](https://www.keycloak.or
 ### Set Up Trusted Hosts
 - In Realm Settings, go to Client Registration and Client Registration Policies. Under 'Anonymous Access Policies', enter 'Trusted Hosts' and add '\*.workshop.nmos.tv' (or similar).
 - Go to the 'Clients' menu and edit the 'admin-cli' client. Under Client Scopes, add the newly defined scope(s) above to the 'Optional client scopes' list. This is useful to enable the debug procedure below.
+- Next, ensure that Keycloak trusts the certificate authority which is in use. This can be achieved by following [Keycloak's instructions](https://www.keycloak.org/docs/6.0/server_installation/#_truststore), or by adding the certificate to the default Java keystore using a command like the following, before restarting Keycloak.
+
+```
+keytool -import -alias nmosca -file cert.pem -cacerts -storepass changeit
+```
```

Comment:
Should we be using the latest link instead of v6.0?

```suggestion
- Next, ensure that Keycloak trusts the certificate authority which is in use. This can be achieved by following [Keycloak's instructions](https://www.keycloak.org/docs/latest/server_installation/#_truststore), or by adding the certificate to the default Java keystore using a command like the following, before restarting Keycloak.

```
keytool -import -alias nmosca -file cert.pem -cacerts -storepass changeit
```
```